locations/spiders/schlotzskys.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import json
+
+from locations.items import GeojsonPointItem
+
+
+class SchlotzskysSpider(scrapy.Spider):
+    name = 'schlotzskys'
+    allowed_domains = ['schlotzskys.com']
+    start_urls = ['https://www.schlotzskys.com/restaurants/']
+
+    def parse_hours(self, hours):
+        hours = hours.split('\t')[6]
+        return hours
+
+    # Left this here for later. Maybe it would be useful eventually.
+##    def parse_lat_lon(self, address):
```

Comment:
We'll geocode using a different method later. Please remove this function for now.